respa_admin/templates/resources/form/_nav.html

```diff
@@ -12,18 +13,12 @@ <h2>
             </div>
 
             <div class="language-switcher">
-                <button name="language-sv" type="submit" value="" class='language-item btn'>
-                    {% trans "Swedish" %}
-                    <span class="badge">5</span>
-                </button>
-                <button name="language-fi" type="submit" value="" class='language-item btn'>
-                    {% trans "Finnish" %}
-                    <span class="badge">5</span>
-                </button>
-                <button name="language-en" type="submit" value="" class='language-item btn'>
-                    {% trans "English" %}
-                    <span class="badge">5</span>
-                </button>
+                {% for key, value in LANGUAGES %}
+                    <button name="language-{{ key }}" value="{{ key }}" class='language-item btn'>
+                        {{ value }}
+                        <span class="badge">{{ trans_fields|get_value_from_dict:key}}</span>
```

Comment:
The badge is supposed to show the number of missing translations as far as I understand. So it should be updated by Javascript when new translations are entered.

E.g. if the form has following data:
```
name_en: Test
name_fi: Testi
name_sv:

description_en: This is a test resource.
description_fi:
description_sv:

specific_terms_en:
specific_terms_fi:
specific_terms_sv:
```
then the badges should show:
 * EN: 0  (no missing translations)
 * FI: 1  (missing = description)
 * SV: 2 (missing = name + description)

The "specific_terms" field is not counted as missing, since it doesn't have value in any language.